From: Jean Delvare <jdelvare@suse.de>
Subject: tulip: Disable debugging messages by default
References: http://lists.opensuse.org/opensuse-kernel/2011-07/msg00060.html
Patch-mainline: Not yet, submitted

With debugging messages enabled by default, one user complained that
his log was spammed with these messages:

[ 6233.528227] dmfe: tdes0=7fff0000
[ 6237.651056] dmfe: tdes0=7fff0000
[ 6237.651060] dmfe: tdes0=7fff0000
[ 6237.696019] dmfe: tdes0=7fff0000
[ 6237.696022] dmfe: tdes0=7fff0000
[ 6238.028810] dmfe: tdes0=7fff0000
[ 6238.028814] dmfe: tdes0=7fff0000
[ 6238.073544] dmfe: tdes0=7fff0000
[ 6238.073547] dmfe: tdes0=7fff0000

With dynamic debugging available, I think it makes no sense to enable
debugging messages by default, especially for such old drivers which
aren't been actively developed any longer.

Signed-off-by: Jean Delvare <jdelvare@suse.de>
Cc: Grant Grundler <grundler@parisc-linux.org>
---
 drivers/net/tulip/Makefile |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/net/tulip/Makefile
+++ b/drivers/net/tulip/Makefile
@@ -2,8 +2,6 @@
 # Makefile for the Linux "Tulip" family network device drivers.
 #
 
-ccflags-$(CONFIG_NET_TULIP)	:= -DDEBUG
-
 obj-$(CONFIG_PCMCIA_XIRCOM)	+= xircom_cb.o
 obj-$(CONFIG_DM9102)		+= dmfe.o
 obj-$(CONFIG_WINBOND_840)	+= winbond-840.o
